client/blocks/stats-navigation/style.scss

```diff
@@ -0,0 +1,51 @@
+.stats-navigation {
+	.section-nav__panel {
+		padding-right: 16px;
+
+		@include breakpoint( "<480px" ) {
```

Comment:
Just a note that there is a preference for mobile first media queries: https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#media-queries